lib/create-custom-runner.ts

```diff
@@ -20,8 +20,10 @@ const createRemoteCache = (
     options
   );
 
-  const read = options.read ?? process.env.NXCACHE_READ !== "false";
-  const write = options.write ?? process.env.NXCACHE_WRITE !== "false";
+  const read = process.env.NXCACHE_READ ?
+    process.env.NXCACHE_READ !== "false" : (options.read || true);
+  const write = process.env.NXCACHE_WRITE ?
+    process.env.NXCACHE_WRITE !== "false" : (options.write || true);
```

Comment:
```suggestion
    process.env.NXCACHE_READ !== "false" : (options.read ?? true);
  const write = process.env.NXCACHE_WRITE ?
    process.env.NXCACHE_WRITE !== "false" : (options.write ?? true);
```

If `options.read` or `options.write` is `false` we want to use the `false`. If those values are `undefined` we default to `true`.

Reply:
Good catch